core/server/models/post.js

```diff
@@ -99,6 +103,17 @@ Post = ghostBookshelf.Model.extend({
         return [...keys, ...postsMetaKeys];
     },
 
+    orderRawAttributes: function orderRawAttributes(withRelated) {
+        if (withRelated && withRelated.indexOf('email') > -1) {
+            return {
+                'email.open_rate': {
+                    orderByRaw: 'emails.opened_count / emails.email_count * 100',
```

Comment:
this `orderByRaw` expression will become just `open_rate` once the column is introduced, correct? In that case we might not even need a special "raw" clause and order by relation just like we do with `post_meta` fields?

Reply:
No, the column was a backup in case the on-the-fly ordering wasn't feasible.